src/__tests__/toPropertyPath.spec.ts

```diff
@@ -7,4 +7,5 @@ test('toPropertyPath', () => {
   expect(toPropertyPath('/paths/~1~1~1~1pets/post')).toEqual('paths.////pets.post');
   expect(toPropertyPath('/paths/foo~0users')).toEqual('paths.foo~users');
   expect(toPropertyPath('#/definitions/Error')).toEqual('definitions.Error');
+  expect(toPropertyPath('/a/..dotted/path')).toEqual('a.["..dotted"].path');
```

Comment:
I'm sure you'll tell it's not worth testing but I would add such check too:

```suggestion
  expect(toPropertyPath('/a/..dotted/path')).toEqual('a.["..dotted"].path');
    expect(toPropertyPath('/a/dot.ted/path')).toEqual('a.["dot.ted"].path');
```

Reply:
I treat that file as unit test (so white-box text) and it is not worth. .indexOf('.') does not care whether the dot is on the beginning or in the middle.

Reply:
How about I change the implementation to `indexOf('.') === 0` and will run your tests again? :P

Reply:
You'll get false positive :P

Reply:
oh in case you say: "yeah we could change the implementation to `indexOf('.') === random()"
yes, that's why I think we should randomize tests :P